src/content/app/genome-browser/components/browser-track-config/BrowserTrackConfig.test.tsx

```diff
@@ -15,40 +15,82 @@
  */
 
 import React from 'react';
-import configureMockStore from 'redux-mock-store';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
 import { render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { Provider } from 'react-redux';
-import thunk from 'redux-thunk';
 
-import { createMockBrowserState } from 'tests/fixtures/browser';
 import MockGenomeBrowser from 'tests/mocks/mockGenomeBrowser';
 
 import * as trackConfigActions from 'src/content/app/genome-browser/state/track-config/trackConfigSlice';
+import * as browserGeneralActions from 'src/content/app/genome-browser/state/browser-general/browserGeneralSlice';
```

Comment:
It's so odd we are calling these two imports "actions". As a result, you write below `browserGeneralActions.defaultBrowserGeneralState`, `trackConfigActions.defaultTrackConfigsForGenome`, `trackConfigActions.TrackType.GENE`, etc., which has nothing to do with actions.

Why don't we call them slices?